.travis.yml

```diff
@@ -1,19 +1,28 @@
 language: dart
 sudo: required
+
 addons:
   chrome: stable
 
 # Re-use downloaded pub packages everywhere.
 cache:
   directories:
     - $HOME/.pub-cache
-
-dart:
-  - 1.24.3
-  - stable
-
-script:
-  - npm install
-  - dartfmt --set-exit-if-changed -n lib test example
-  - dartanalyzer lib test example
-  - tool/test.sh
+jobs:
+  include:
+    - dart: 2.4.0
```

Comment:
should we update this to 2.6.1 while we're at it?

Reply:
`stable` version will run on 2.6.1. This way we running on lowest version version too.

Reply:
https://travis-ci.org/Workiva/opentracing_dart/jobs/615227783#L197